qvm/template-gui.jinja

```diff
@@ -12,8 +12,11 @@
   file.managed:
     - name: /etc/qubes/policy.d/50-gui-{{ vmname }}.policy
     - contents: |
+        policy.EvalGUI        +ClipboardPaste   {{ vmname }}             dom0                             allow
```

Comment:
```suggestion
        policy.EvalGUI        +qubes.ClipboardPaste   {{ vmname }}             dom0                             allow
```

Reply:
Oops!  Looks like I made this mistake *everywhere*.